frontend/public/components/pod.tsx

```diff
@@ -104,140 +109,209 @@ const tableColumnClasses = [
         transforms: [sortable],
         props: { className: tableColumnClasses[5] },
+        visible: true,
       },
       {
         title: 'Memory',
+        ID: 'MEMORY',
         sortFunc: 'podMemory',
         transforms: [sortable],
         props: { className: tableColumnClasses[6] },
+        visible: true,
       },
       {
         title: 'CPU',
+        ID: 'CPU',
         sortFunc: 'podCPU',
         transforms: [sortable],
         props: { className: tableColumnClasses[7] },
+        visible: true,
       },
       {
         title: 'Created',
+        ID: 'CREATED',
         sortField: 'metadata.creationTimestamp',
         transforms: [sortable],
         props: { className: tableColumnClasses[8] },
+        visible: true,
       },
       {
-        title: '',
+        title: 'Node',
+        ID: 'NODE',
+        sortField: 'spec.nodeName',
+        transforms: [sortable],
         props: { className: tableColumnClasses[9] },
+        visible: false,
+        additional: true,
+      },
+      {
+        title: 'Labels',
+        ID: 'LABELS',
+        sortField: 'metadata.labels',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[10] },
+        visible: false,
+        additional: true,
+      },
+      {
+        title: 'IP Address',
+        ID: 'IP-ADDRESS',
+        sortField: 'status.hostIP',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[11] },
+        visible: false,
+        additional: true,
+      },
+      {
+        title: '',
+        props: { className: tableColumnClasses[12] },
+        visible: true,
       },
     ];
   };
 };
 
+const PodTableRow = connect<PodTableRowPropsFromState, null, PodTableRowProps>(podRowStateToProps)(
+  ({
+    obj: pod,
+    index,
+    rowKey,
+    style,
+    metrics,
+    showNodes,
+    selectedColumns,
+  }: PodTableRowProps & PodTableRowPropsFromState) => {
+    const { name, namespace, creationTimestamp, labels } = pod.metadata;
+    const { readyCount, totalContainers } = podReadiness(pod);
+    const phase = podPhase(pod);
+    const restarts = podRestarts(pod);
+    const bytes: number = _.get(metrics, ['memory', namespace, name]);
+    const cores: number = _.get(metrics, ['cpu', namespace, name]);
+    const columns = selectedColumns?.get(columnManagementID) || getHeader(showNodes)();
+    return (
+      <TableRow id={pod.metadata.uid} index={index} trKey={rowKey} style={style}>
+        <TableData className={tableColumnClasses[0]}>
+          <ResourceLink kind={kind} name={name} namespace={namespace} />
+        </TableData>
+        <TableData
+          className={classNames(tableColumnClasses[1], 'co-break-word')}
+          columns={columns}
+          columnID="NAMESPACE"
+        >
+          <ResourceLink kind="Namespace" name={namespace} />
+        </TableData>
+        <TableData className={tableColumnClasses[2]} columns={columns} columnID="STATUS">
```

Comment:
It would be good to find a way to avoid repeating the IDs for every column. I'm worried there will be typos and it will be hard to maintain.

Reply:
created an object to store repetitive column data (col classes, id's, and titles). reference that for each reference of these properties.